lib/readline.js

```diff
@@ -484,7 +483,7 @@ Interface.prototype._tabComplete = function(lastKeypressWasTab) {
     self.resume();
 
     if (err) {
-      debug('tab completion error %j', err);
```

Comment:
I would rather switch to `this._writeToOutput()`.

Reply:
But it's actually error, and also recommend using `console.error` in https://nodejs.org/api/util.html#util_util_debug_string. Do I miss something ?

Reply:
`readline`/`repl` can be used in non-terminal settings, such as over a socket. Outputting to stderr for a repl being used over a TCP socket would mean the other end still would not see the error. Writing the error message to the output channel would work in all cases.

Reply:
Thanks for the explanation, I will change it.